2024-01-31 21:30:48,790 INFO    StreamThr :65832 [internal.py:wandb_internal():86] W&B internal server running at pid: 65832, started at: 2024-01-31 21:30:48.788596
2024-01-31 21:30:48,792 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: status
2024-01-31 21:30:48,795 INFO    WriterThread:65832 [datastore.py:open_for_write():87] open: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\run-x8cayvyn.wandb
2024-01-31 21:30:48,796 DEBUG   SenderThread:65832 [sender.py:send():382] send: header
2024-01-31 21:30:48,845 DEBUG   SenderThread:65832 [sender.py:send():382] send: run
2024-01-31 21:30:49,655 INFO    SenderThread:65832 [dir_watcher.py:__init__():211] watching files in: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files
2024-01-31 21:30:49,656 INFO    SenderThread:65832 [sender.py:_start_run_threads():1136] run started: x8cayvyn with start time 1706754648.790911
2024-01-31 21:30:49,663 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: check_version
2024-01-31 21:30:49,663 DEBUG   SenderThread:65832 [sender.py:send_request():409] send_request: check_version
2024-01-31 21:30:50,454 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: run_start
2024-01-31 21:30:52,174 DEBUG   HandlerThread:65832 [system_info.py:__init__():27] System info init
2024-01-31 21:30:52,174 DEBUG   HandlerThread:65832 [system_info.py:__init__():42] System info init done
2024-01-31 21:30:52,174 INFO    HandlerThread:65832 [system_monitor.py:start():194] Starting system monitor
2024-01-31 21:30:52,174 INFO    SystemMonitor:65832 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-01-31 21:30:52,174 INFO    HandlerThread:65832 [system_monitor.py:probe():214] Collecting system info
2024-01-31 21:30:52,190 INFO    SystemMonitor:65832 [interfaces.py:start():190] Started cpu monitoring
2024-01-31 21:30:52,198 INFO    SystemMonitor:65832 [interfaces.py:start():190] Started disk monitoring
2024-01-31 21:30:52,198 INFO    SystemMonitor:65832 [interfaces.py:start():190] Started gpu monitoring
2024-01-31 21:30:52,198 INFO    SystemMonitor:65832 [interfaces.py:start():190] Started memory monitoring
2024-01-31 21:30:52,210 INFO    SystemMonitor:65832 [interfaces.py:start():190] Started network monitoring
2024-01-31 21:30:52,220 DEBUG   HandlerThread:65832 [system_info.py:probe():151] Probing system
2024-01-31 21:30:52,220 DEBUG   HandlerThread:65832 [system_info.py:_probe_git():136] Probing git
2024-01-31 21:30:52,339 DEBUG   HandlerThread:65832 [system_info.py:_probe_git():144] Probing git done
2024-01-31 21:30:52,339 DEBUG   HandlerThread:65832 [system_info.py:probe():199] Probing system done
2024-01-31 21:30:52,339 DEBUG   HandlerThread:65832 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.22621-SP0', 'python': '3.11.5', 'heartbeatAt': '2024-02-01T02:30:52.220110', 'startedAt': '2024-02-01T02:30:48.780506', 'docker': None, 'cuda': None, 'args': ('-c', 'bc'), 'state': 'running', 'program': 'C:\\Users\\tomda\\Desktop\\symmetry_learning\\symmetry_shift_runner.py', 'codePathLocal': 'symmetry_shift_runner.py', 'codePath': 'symmetry_shift_runner.py', 'git': {'remote': 'git@github.com:tkaminsky/symmetry_learning.git', 'commit': 'ffba65af91d1ba4999bc200beca3888cc6613e10'}, 'email': 'tomdavkam@gmail.com', 'root': 'C:/Users/tomda/Desktop/symmetry_learning', 'host': 'tomdavkam', 'username': 'tomda', 'executable': 'C:\\Users\\tomda\\anaconda3\\python.exe', 'cpu_count': 8, 'cpu_count_logical': 16, 'cpu_freq': {'current': 2470.0, 'min': 0.0, 'max': 4001.0}, 'cpu_freq_per_core': [{'current': 2470.0, 'min': 0.0, 'max': 4001.0}], 'disk': {'/': {'total': 794.588981628418, 'used': 620.6595115661621}}, 'gpu': 'NVIDIA GeForce RTX 4070 Laptop GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4070 Laptop GPU', 'memory_total': 8585740288}], 'memory': {'total': 31.21628189086914}}
2024-01-31 21:30:52,339 INFO    HandlerThread:65832 [system_monitor.py:probe():224] Finished collecting system info
2024-01-31 21:30:52,339 INFO    HandlerThread:65832 [system_monitor.py:probe():227] Publishing system info
2024-01-31 21:30:52,339 DEBUG   HandlerThread:65832 [system_info.py:_save_conda():208] Saving list of conda packages installed into the current environment
2024-01-31 21:30:52,681 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\conda-environment.yaml
2024-01-31 21:30:54,232 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:30:54,232 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:30:54,644 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:30:54,644 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:30:56,654 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:30:56,654 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:30:57,053 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:30:57,053 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:30:59,070 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:30:59,070 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:30:59,465 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:30:59,465 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:31:01,470 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:31:01,470 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:31:01,875 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:31:01,875 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:31:03,096 DEBUG   HandlerThread:65832 [system_info.py:_save_conda():220] Saving conda packages done
2024-01-31 21:31:03,107 INFO    HandlerThread:65832 [system_monitor.py:probe():229] Finished publishing system info
2024-01-31 21:31:03,112 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: status_report
2024-01-31 21:31:03,112 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: keepalive
2024-01-31 21:31:03,112 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: status_report
2024-01-31 21:31:03,112 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: keepalive
2024-01-31 21:31:03,112 DEBUG   SenderThread:65832 [sender.py:send():382] send: files
2024-01-31 21:31:03,112 INFO    SenderThread:65832 [sender.py:_save_file():1403] saving file wandb-metadata.json with policy now
2024-01-31 21:31:03,120 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: python_packages
2024-01-31 21:31:03,120 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: stop_status
2024-01-31 21:31:03,120 DEBUG   SenderThread:65832 [sender.py:send_request():409] send_request: python_packages
2024-01-31 21:31:03,120 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-31 21:31:03,120 DEBUG   SenderThread:65832 [sender.py:send_request():409] send_request: stop_status
2024-01-31 21:31:03,388 DEBUG   SenderThread:65832 [sender.py:send():382] send: telemetry
2024-01-31 21:31:03,773 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\conda-environment.yaml
2024-01-31 21:31:03,774 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\output.log
2024-01-31 21:31:03,774 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\requirements.txt
2024-01-31 21:31:03,774 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\wandb-metadata.json
2024-01-31 21:31:03,883 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:31:03,884 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:31:04,292 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:31:04,292 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:31:04,589 INFO    wandb-upload_0:65832 [upload_job.py:push():131] Uploaded file C:\Users\tomda\AppData\Local\Temp\tmpct2f6bepwandb\dxi1hkah-wandb-metadata.json
2024-01-31 21:31:04,660 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: status_report
2024-01-31 21:31:05,799 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\output.log
2024-01-31 21:31:06,302 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:31:06,303 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:31:06,715 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:31:06,716 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:31:07,811 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\output.log
2024-01-31 21:31:08,730 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:31:08,730 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:31:09,146 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:31:09,146 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:31:09,796 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: status_report
2024-01-31 21:31:09,828 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\output.log
2024-01-31 21:31:11,155 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:31:11,155 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:31:11,605 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:31:11,606 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:31:11,845 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\output.log
2024-01-31 21:31:13,861 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\output.log
2024-01-31 21:31:15,586 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: status_report
2024-01-31 21:31:15,886 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\output.log
2024-01-31 21:31:17,894 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\output.log
2024-01-31 21:31:18,130 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-31 21:31:18,138 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: stop_status
2024-01-31 21:31:18,138 DEBUG   SenderThread:65832 [sender.py:send_request():409] send_request: stop_status
2024-01-31 21:31:19,911 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\output.log
2024-01-31 21:31:21,473 DEBUG   HandlerThread:65832 [handler.py:handle_request():146] handle_request: status_report
2024-01-31 21:31:21,925 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\output.log
2024-01-31 21:31:21,925 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\config.yaml
2024-01-31 21:31:23,669 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:31:23,669 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-01-31 21:31:23,949 INFO    Thread-16 :65832 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\tomda\Desktop\symmetry_learning\wandb\run-20240131_213048-x8cayvyn\files\output.log
2024-01-31 21:31:24,144 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:31:24,144 ERROR   gpu       :65832 [interfaces.py:monitor():144] Failed to sample metric: No data
2024-01-31 21:31:24,314 INFO    memory    :65832 [interfaces.py:monitor():140] Process proc.memory.rssMB has exited.
2024-01-31 21:31:24,322 DEBUG   SystemMonitor:65832 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-01-31 21:31:24,338 DEBUG   SystemMonitor:65832 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-01-31 21:31:24,354 DEBUG   SystemMonitor:65832 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-01-31 21:31:24,387 DEBUG   SenderThread:65832 [sender.py:send():382] send: stats
2024-01-31 21:31:25,765 INFO    MainThread:65832 [internal.py:handle_exit():76] Internal process exited
